modular-docs-manual/content/topics/module_auditing.adoc

```diff
@@ -0,0 +1,18 @@
+[[auditing]]
+= Auditing Your Feature-Based Manual
+
+In the process of adding content to the modules from your existing feature-based manual, it is likely that there is some existing content that did not fit into any of the user stories that you identified and thus was not pulled out. It is important to identify this content to ensure that it is no longer needed.
+
+.Procedure
+
+. Go through your existing feature-based manual and identify any content that you did not add to an assembly or module.
+
+. For each piece of content that you find, determine whether it is necessary.
++
+--
+If the content does not fit into a user story or assembly, then it stands to reason that it may not be necessary to complete any particular user goal. If this is the case, you should be able to discard it.
+
+For example, many feature-based manuals contain detailed sections about how a particular feature works. This type of content, which valuable in certain contexts, is not always necessary for user documentation -- user goals generally involve _doing_ something, not understanding how all the details work. When you encounter this type of content, ask yourself whether a user would need to understand it in order to accomplish any of their goals (planning, installing, configuring, managing, and so on). If the answer is no, then you can probably discard it.
```

Comment:
Can you edit this into an example box too? It would help separate the example from the rest of the step, just like in the previous step.

Reply:
Done.